pkg/job/runtime_v2/kubernetes_runtime.go

```diff
@@ -518,11 +519,45 @@ func (kr *KubeRuntime) createPersistentVolumeClaim(namespace string, pvc *corev1
 	return kr.clientset().CoreV1().PersistentVolumeClaims(namespace).Create(context.TODO(), pvc, metav1.CreateOptions{})
 }
 
+func (kr *KubeRuntime) GetPersistentVolumeClaims(namespace, name string, getOptions metav1.GetOptions) (*corev1.PersistentVolumeClaim, error) {
+	return kr.clientset().CoreV1().PersistentVolumeClaims(namespace).Get(context.TODO(), name, getOptions)
+}
+
 func (kr *KubeRuntime) DeletePersistentVolumeClaim(namespace string, name string,
 	deleteOptions metav1.DeleteOptions) error {
 	return kr.clientset().CoreV1().PersistentVolumeClaims(namespace).Delete(context.TODO(), name, deleteOptions)
 }
 
+type PatchMapValue struct {
```

Comment:
It might be better to define `PatchMapValue` in  `PatchPVCFinalizerNull` function.
```golang
payload := []struct{
	Op    string   `json:"op"`
	Path  string   `json:"path"`
	Value []string `json:"value"`
}{
  {
     	Op:    "replace",
       Path:  "/metadata/finalizers",
	Value: nil,
  }
}
```